CMakeLists.txt

```diff
@@ -29,7 +29,12 @@ else ()
   add_library(netflow9 STATIC ${CXX_SRC})
 endif ()
 
-target_include_directories(netflow9 SYSTEM PUBLIC include)
+get_property(SRC_DIR DIRECTORY PROPERTY SOURCE_DIR)
+message(STATUS "tins include:" ${SRC_DIR}/external/libtins/include/)
```

Comment:
Looks like you left this line for debugging. It's pretty useless information - it does not depend on anything external, and will always be the same once you download the repository, there is no way to change it.